datacite-delete-draft-doi-handler/src/main/java/no/unit/nva/datacite/handlers/resource/ExternalUpdatesEventHandler.java

```diff
@@ -0,0 +1,136 @@
+package no.unit.nva.datacite.handlers.resource;
+
+import static java.util.Objects.nonNull;
+import static nva.commons.core.attempt.Try.attempt;
+import com.amazonaws.services.lambda.runtime.Context;
+import com.amazonaws.services.lambda.runtime.RequestHandler;
+import com.amazonaws.services.lambda.runtime.events.SQSEvent;
+import com.amazonaws.services.lambda.runtime.events.SQSEvent.SQSMessage;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+import no.unit.nva.commons.json.JsonUtils;
+import no.unit.nva.doi.DoiClient;
+import no.unit.nva.doi.datacite.clients.DataCiteClientV2;
+import no.unit.nva.doi.datacite.clients.exception.ClientException;
+import no.unit.nva.doi.datacite.restclient.models.State;
+import no.unit.nva.doi.models.Doi;
+import no.unit.nva.events.models.AwsEventBridgeDetail;
+import no.unit.nva.events.models.AwsEventBridgeEvent;
+import no.unit.nva.events.models.EventReference;
+import no.unit.nva.s3.S3Driver;
+import nva.commons.core.Environment;
+import nva.commons.core.JacocoGenerated;
+import nva.commons.core.attempt.Failure;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import software.amazon.awssdk.services.s3.S3Client;
+
+public class ExternalUpdatesEventHandler implements RequestHandler<SQSEvent, Void> {
```

Comment:
The plan is to handle more deletions in this handler? Or will this handler delete draft dois only?